Automattic/jetpack
# Phan: Added internal stub loading capability to the config base.

## Proposed changes:

* Adds a base config parameter to be able to include Phan provided [internal stubs](https://github.com/phan/phan/wiki/How-To-Use-Stubs#internal-stubs).

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:





* Make sure Phan works as before.
* Include code that uses classes provided by PHP extensions like `\ZipArchive` or `SQLite3`, see that Phan doesn't barf at undefined class.
